### ISSUE TITLE
Kivymd not working

### ISSUE THREAD

**Issue by GPereira121** [kivymd/KivyMD#1517]
### Description of the Bug

Your text

### Code and Logs

```python
from kivy.app import App
from kivy.lang import Builder

kv = """
Screen:
    # KV-Code
"""


class MainApp(App):
    def build(self):
        self.root = Builder.load_string(kv)


if __name__ == '__main__':
    MainApp().run()
```

### Screenshots

Add images to explain us this bug. Paste urls here.

Remove this section if no images here

### Versions

* OS: 
* Python: 
* Kivy: 
* KivyMD:

**Comment by GPereira121** [comment]
Thanks for your prompt response. I cannot get kivymd to work. It is installed in a virtual environment and i can see the files. But i still get a message saying no module named kivymd. 


Sent from Yahoo Mail for iPhone


On Wednesday, June 28, 2023, 3:55 PM, Yuri Ivanov ***@***.***> wrote:




Closed #1517 as completed.

**Comment by GPereira121** [comment]
Here is the dir of kivy from my file explorer packages\kivymd

Mode                 LastWriteTime         Length Name----                 -------------         ------ ----d-----         6/27/2023  12:20 PM                datad-----         6/27/2023  12:20 PM                effectsd-----         6/27/2023  12:20 PM                fontsd-----         6/27/2023  12:20 PM                imagesd-----         6/27/2023  12:20 PM                toastd-----         6/27/2023  12:20 PM                toolsd-----         6/27/2023  12:20 PM                uixd-----         6/27/2023  12:20 PM                utilsd-----         6/27/2023  12:20 PM                _pycache_-a----         6/27/2023  12:20 PM           4188 app.py-a----         6/27/2023  12:20 PM          21550 color_definitions.py-a----         6/27/2023  12:20 PM           6234 factory_registers.py-a----         6/27/2023  12:20 PM           1628 font_definitions.py-a----         6/27/2023  12:20 PM         265961 icon_definitions.py-a----         6/27/2023  12:20 PM           1035 material_resources.py-a----         6/27/2023  12:20 PM          57563 theming.py-a----         6/27/2023  12:20 PM           2920 theming_dynamic_text.py-a----         6/27/2023  12:20 PM            145 _version.py-a----         6/27/2023  12:20 PM           1973 _init_.py

PS C:\users\g_per\documents\myenv\lib\site-packages\kivymd>
Not sure what is wrong. 
    On Wednesday, June 28, 2023, 03:55:44 PM EDT, Yuri Ivanov ***@***.***> wrote:  
 
 


Closed #1517 as completed.
